src/evaluators/SwitchStatement.js

```diff
@@ -33,6 +33,19 @@ function CaseSelectorEvaluation(
   return Environment.GetValue(realm, exprRef);
 }
 
+function AbstractCaseBlockEvaluation(
+  cases: Array<BabelNodeSwitchCase>,
+  defaultCaseNum: number,
+  input: Value,
+  strictCode: boolean,
+  env: LexicalEnvironment,
+  realm: Realm
+): Value {
+  invariant(realm.useAbstractInterpretation);
+
+  invariant(false, "Can't do abstract interpretation of switch blocks yet.");
```

Comment:
Rather treat this as user error than as an internal Prepack failure:

AbstractValue.reportIntrospectionError(input);
throw new FatalError();

Reply:
Ah, thanks! I was wondering the right way to emit an error like that. `invariant(false)` felt a little hacky.